specification/ml/get_calendars/MlGetCalendarsRequest.ts

```diff
@@ -23,22 +23,29 @@ import { Id } from '@_types/common'
 import { integer } from '@_types/Numeric'
 
 /**
+ * Retrieves configuration information for calendars.
  * @rest_spec_name ml.get_calendars
  * @since 6.2.0
  * @stability stable
+ * @cluster_privileges monitor_ml
  */
 export interface Request extends RequestBase {
   path_parts: {
-    /** A string that uniquely identifies a calendar. */
+    /** A string that uniquely identifies a calendar. You can get information for multiple calendars by using a comma-separated list of ids or a wildcard expression. You can get information for all calendars by using `_all` or `*` or by omitting the calendar identifier.*/
     calendar_id?: Id
   }
   query_parameters: {
-    /** Skips the specified number of calendars. */
+    /** Skips the specified number of calendars. This parameter is supported only when you omit the calendar identifier.
+     * @server_default 0
+     */
     from?: integer
-    /** Specifies the maximum number of calendars to obtain. */
+    /** Specifies the maximum number of calendars to obtain. This parameter is supported only when you omit the calendar identifier.
+     * @server_default 10000
+     */
     size?: integer
   }
   body: {
```

Comment:
Given the decision in https://github.com/elastic/elasticsearch/issues/79271 should we omit the `body` in this API as well?

Reply:
This particular API didn't fail when I ran the `make contrib`, so I thought it was not an exceptional one.  It has both "GET" and "POST" in its spec here: https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/ml.get_calendars.json

Reply:
@lcawl You're right, I must have been looking at the wrong spec.